doc/api/process.md

```diff
@@ -850,8 +850,8 @@ added: v0.1.13
 
 * `code` {Integer} The exit code. Defaults to `0`.
 
-The `process.exit()` method instructs Node.js to terminate the process as
-quickly as possible with the specified exit `code`. If the `code` is omitted,
+The `process.exit()` method instructs Node.js to terminate the process
+synchronously with the specified exit `code`. If the `code` is omitted,
```

Comment:
Maybe we should add something like "the exit is immediate except that synchronous code may still be run from the `exit` event"?